components/nav/Header.vue

```diff
@@ -128,6 +128,9 @@ export default {
               </div>
             </li>
             <div>
+              <nuxt-link tag="li" :to="{name: 'account'}" class="hand">
+                <a>API Keys</a>
+              </nuxt-link>
```

Comment:
@vincent99 This was a place holder - API keys will be on the same page as he account info (where Richard added the Change password). I guess this will need to show 'Account' if not multi cluster and 'Account & API Keys' if multi cluster ? and we'll need to hide the API Keys from the account page - OR should both the Account and API Keys functionality be hidden?

Reply:
At least for the moment multi-cluster and multi-user are tied together so you can just hide the whole nav link here if !isMultiCluster.